app/components/user_mentee_application/list_item_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class UserMenteeApplication::ListItemComponent < ViewComponent::Base
+  def initialize(mentee_application:)
+    @mentee_application = mentee_application
+  end
+
+  private
+
+  attr_reader :mentee_application
+
+  def link_text
+    application_cohort = mentee_application.user_mentee_application_cohort
+
+    "Application for #{application_cohort&.name || 'The Agency of Learning'}"
```

Comment:
Have to occasionally do these contortions to account for a possible null cohort. Would maybe be cool to have cohorts be mandatory for any application, but I can't remember if that effects some of the other ideas around this feature.

Reply:
cohorts are mandatory for any application (or should be)

Reply:
Ok just checked and we have backfilled everything. Feel free to make it `null: false` in this PR.